includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -615,8 +615,8 @@ public function init_settings () {
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_EDITOR_TOOLBAR_CLEAN )
-				          ->set_title( __( 'Clean formatting', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Show clean formatting option in editor toolbar.', "anycomment" ) ) ),
+				          ->set_title( __( 'Clear formatting', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later